### PR TITLE
use surface material

### PR DESCRIPTION
**Pull request by dasg34**

### CODE REVIEW

File: MREGodotRuntimeLib/App/MixedRealityExtensionApp.cs [hunk 1]
```diff
@@ -1013,9 +1013,9 @@ void ProcessActors(Spatial node3D, Actor parent)
 				if (actor.MeshInstance != null)
 				{
 					// only overwrite material if there's something in the cache, i.e. not a random library material
-					if (actor.MeshInstance.MaterialOverride != null)
 					{
```
Comment (woohyun) [inline MREGodotRuntimeLib/App/MixedRealityExtensionApp.cs:1016]: I think this is not needed anymore :)

File: MREGodotRuntimeLib/App/MixedRealityExtensionApp.cs [hunk 2]
```diff
@@ -1013,9 +1013,9 @@ void ProcessActors(Spatial node3D, Actor parent)
...
-						var matId = AssetManager.GetByObject(actor.MeshInstance.MaterialOverride)?.Id;
+						var material = actor.MeshInstance.GetSurfaceMaterial(0);
```
Comment (woohyun) [inline MREGodotRuntimeLib/App/MixedRealityExtensionApp.cs:1017]: Just a question.
What is the difference between MaterialOverride and GetSurfaceMaterial ?

Comment (dasg34) [reply MREGodotRuntimeLib/App/MixedRealityExtensionApp.cs:1017]: When creating a prefab actor, a mesh can have multiple materials. if the mesh only has one material, `get_surface_material(0)` is same as material_override.

**Review by woohyun — APPROVED** [review]
LGTM

**Pull request merged**